graphql-dgs/src/test/kotlin/com/netflix/graphql/dgs/internal/DefaultDgsQueryExecutorTest.kt

```diff
@@ -204,6 +205,17 @@ internal class DefaultDgsQueryExecutorTest {
             .isInstanceOf(InvalidSyntaxError::class.java)
     }
 
+    @Test
+    fun `Invalid operation returns a GraphQL Error with type UnknownOperationException`() {
```

Comment:
Should we also check  that the status is set to BAD_REQUEST?

Reply:
I've added an assertion for that.